sites/partners/src/listings/PaperListingForm/sections/ListingPhoto.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from "react"
+import { useFormContext } from "react-hook-form"
+import {
+  cloudinaryUrlFromId,
+  t,
+  AppearanceStyleType,
+  AppearanceBorderType,
+  Button,
+  Dropzone,
+  CloudinaryUpload,
+  Drawer,
+  GridSection,
+  GridCell,
+  MinimalTable,
+  TableThumbnail,
+} from "@bloom-housing/ui-components"
+import { AssetsService } from "@bloom-housing/backend-core/types"
+
+let newCloudinaryId = ""
+
+const ListingPhoto = () => {
+  const formMethods = useFormContext()
+
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, setValue, watch } = formMethods
+
+  const cloudName = process.env.cloudinaryCloudName
+  const uploadPreset = process.env.cloudinarySignedPreset
+
+  const listingFormPhoto = watch("image")
+
+  const [drawerState, setDrawerState] = useState(false)
+  const [progressValue, setProgressValue] = React.useState(0)
+  const [cloudinaryImage, setCloudinaryImage] = React.useState("")
+  const resetDrawerState = () => {
+    newCloudinaryId = ""
+    setProgressValue(0)
+    setCloudinaryImage("")
```

Comment:
So we're consistent in how we're setting variables, can you update it so that newCloudinaryId uses useState. It looks like it'd be safe to have cloudinary image and id together in one object and updated together.